spec/unit/utils/encryptable_move_handler_spec.rb

```diff
@@ -90,7 +90,7 @@
 
 
     decrypted = credential.decrypt(new_folder.team.decrypt_team_password(bob, private_key))
-    expect(decrypted).to eq('password')
+    expect(decrypted).to eq({:label=>"testcustomattrlabel", :value=>"testcustomattrvalue"})
```

Comment:
Custom Attribute is the last value that is decrypted, before the custom attr feature it was the password, we now just check for the encrypted value of custom attr instead of password